frontend/src/lib/components/navigation/Navigation.svelte

```diff
@@ -0,0 +1,72 @@
+<script lang="ts">
+  import {Icon} from '$lib/components/icon';
+  import type {NavigationProps} from './Navigation.type';
+
+  type $$Props = NavigationProps;
+
+  export let items: $$Props['items'];
+</script>
+
+<!--
+@component
+Outputs the navigation items for the application.
+
+### Properties
+
+- `items`: An array of arrays the navigation items containing a `label`, 
+  `href` or `onClick` and an optional name of the `icon` for each item. 
+  The sub-arrays are displayed as groups of navigation items.
+- Any valid attributes of a `nav` element.
+
+### Usage
+
+```tsx
+<script lang="ts">
+  import {navigationItems} from '$voter/navigation';
+</script>
+
+<Navigation items={$navigationItems} aria-label="Main navigation" />
+```
+-->
+
+<nav {...$$restProps}>
```

Comment:
I may have a big suggestion 😅: I think, we should have two components, the `Navigation` component, and a `NavItem` component. Something like this:

```Svelte
<script>
	import { Navbar, NavbarItem } from 'yesvelte'
</script>

<Navivation>
	<NavItem title="Home" />
	<NavItem title="Features" />
	<NavItem title="Pricing" />
	<NavItem title="Disabled" disabled />
</Navigation>
```

This way we can assign classes and the `aria` properties to each navigation Item. With the current implementation, it is not possible to pass properties to the navigation Items. 

Some inspiration:
https://mui.com/material-ui/react-menu/
https://www.yesvelte.com/docs/navbar

Reply:
https://joyofcode.xyz/svelte-context-with-stores

Reply:
I'll convert it to multiple components! I think I'll also need a `NavGroup` which is basically a glorified `ul` :smile